contracts/domain/BosonTypes.sol

```diff
@@ -65,7 +65,8 @@ contract BosonTypes {
         Exchange,
         Funds,
         RaiseDispute,
-        ResolveDispute
+        ResolveDispute,
+        CommitToConditionalOffer
```

Comment:
You could actually replace `RaiseDispute` value with `CommitToConditionalOffer`.

`RaiseDispute` is not an actual type anymore, but we just kept it in for consistency of other values.
But now we have a meaningful replacement so we can get rid of it.

Reply:
done